src/LibraryManager/Manifest.cs

```diff
@@ -121,12 +121,295 @@ private static bool IsValidManifestVersion(string version)
             {
                 return SupportedVersions.Contains(new Version(version));
             }
-            catch(Exception)
+            catch (Exception)
             {
                 return false;
             }
         }
 
+        /// <summary>
+        /// Installs a library with the given libraryId
+        /// </summary>
+        /// <param name="libraryId"></param>
+        /// <param name="providerId"></param>
+        /// <param name="files"></param>
+        /// <param name="destination"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<ILibraryInstallationResult> InstallLibraryAsync(string libraryId, string providerId, IReadOnlyList<string> files, string destination, CancellationToken cancellationToken)
```

Comment:
Can't you just make RestoreLibraryAsync public? This way we have consistency

Reply:
This is to add a new library to the manifest. 
RestoreAsync restores the ones that it finds in the file (all of them).